handlers/amazonec2_test.go

```diff
@@ -67,8 +67,8 @@ func setup(accessKey string, secretKey string, vpcId string, zone string) {
 		return machine, nil
 	}
 
-	getRegistrationUrl = func(accountId string, apiClient *client.RancherClient) (string, error) {
-		return "http://1.2.3.4/v1", nil
+	getRegistrationUrlAndImage = func(accountId string, apiClient *client.RancherClient) (string, string, string, error) {
+		return "http://1.2.3.4/v1", "rancher/agent", "latest", nil
```

Comment:
Id rather see the test use a specific version. v0.7.6 is what is currently set in cattle, can you use that?